src/libecalc/core/consumers/consumer_system.py

```diff
@@ -106,65 +98,51 @@ def _get_stream_conditions_adjusted_for_crossover(
 
         return adjusted_stream_conditions
 
-    def evaluate(
+    def evaluator(
```

Comment:
methods usually have a verb, unless they return a class or an instance? Not sure why/how this is better than `evaluate`? I guess you want to differentiate since `evaluate` is taken, and since it does not inherit/implement from `baseConsumer` any more?